### DISCUSSION TITLE
Send Help

### DISCUSSION THREAD

**Discussion by MaritettaKatsarou** [merkourisa/Augmented-Reality#15] [Q&A]
Καλησπέρα, δεν ξέρω αν το έχει κανείς άλλος αυτό το πρόβλημα αλλά...
ενώ χτες μου έτρεχε μια χαρά ο κωδικας και τα σχήματα και όλα, σήμερα μου τα δείχνει όλα αλλοιωμένα και με εντελώς άλλα χρώματα. Το δοκίμασα από 2 υπολογιστές και μάλιστα κάθε φορα άλλαζε και τα χρώματα χωρίς να έχω κάνει εγώ κάποια αλλαγή.

**Comment by merkourisa** [comment]
@MaritettaKatsarou check [this](https://github.com/merkourisa/Augmented-Reality/discussions/13#discussioncomment-2414018)
